atd-vze/src/queries/tempRecords.js

```diff
@@ -3,14 +3,49 @@ import { gql } from "apollo-boost";
 export const GET_TEMP_RECORDS = gql`
   query getTempRecords {
     crashes(
-      where: { is_temp_record: { _eq: true } }
-      order_by: { record_locator: desc }
+      where: {
+        is_temp_record: { _eq: true }
+        _and: { is_deleted: { _eq: false } }
+      }
     ) {
+      id
       record_locator
       case_id
       crash_timestamp
       updated_by
       updated_at
+      units {
+        id
+      }
+    }
+  }
+`;
+
+export const SOFT_DELETE_TEMP_RECORDS = gql`
+  mutation softDeleteTempRecords($recordId: Int!, $updatedBy: String!) {
+    update_crashes_cris_by_pk(
+      pk_columns: { id: $recordId }
+      _set: { is_deleted: true, updated_by: $updatedBy }
+    ) {
+      id
+    }
+    update_units_cris(
+      where: { crash_id: { _eq: $recordId } }
+      _set: { is_deleted: true, updated_by: $updatedBy }
+    ) {
+      returning {
+        id
+        crash_id
+      }
+    }
+    update_people_cris(
+      where: { units_cri: { crash_id: { _eq: $recordId } } }
```

Comment:
awesome to see this nested `where`! i'm going to rename the relationship `units_cri` to `units_cris` in a forthcoming PR that branches from here. that's an artifact of Hasura trying to guess at pluralization 🤖

Reply:
i was gonna point out the cri thing and ask if we should rename but i forgot, glad you pointed it out!

Reply:
multiple cri = cris. this whole thing is hilarious to me. Thanks for the laugh, Hasura! 😄 And, thanks y'all for fixing. 🚀